crates/modelardb_compression/src/models/swing.rs

```diff
@@ -712,49 +715,50 @@ mod tests {
         fit_all_values
     }
 
-    // fn can_minimize_mse(error_bound: ErrorBound, values: &[Value]) -> bool {
-    //     let mut model_type = Swing::new(error_bound);
-    //     let mut timestamp = START_TIME;
-    //     // Fit model of type Swing to perfectly linear sequence.
-    //     let end_time = START_TIME + values.len() as i64 * SAMPLING_INTERVAL;
-    //     for value in values {
-    //         assert!(model_type.fit_data_point(timestamp, *value));
-    //         timestamp += SAMPLING_INTERVAL;
-    //     }
-    //     // Compute the slope and intercept according to a default case in the Swing and Slide paper
-    //     let (default_slope, default_intercept) = compute_slope_and_intercept(
-    //         model_type.start_time,
-    //         model_type.first_value,
-    //         end_time,
-    //         *values.last().unwrap() as f64,
-    //     );
-    //     // Compute slope and intercept with minimum MSE
-    //     let (first_value, last_value) = model_type.model();
-    //     let (slope_with_minimum_mse, intercept_with_minimum_mse) = compute_slope_and_intercept(
-    //         START_TIME,
-    //         first_value as f64,
-    //         end_time,
-    //         last_value as f64,
-    //     );
-    //     // Compute MSE for each approximated value using: (1) default slope and intercept;
-    //     // (2) slope and intercept with minimum MSE
-    //     let mut mse = 0.0;
-    //     let mut optimized_mse = 0.0;
-
-    //     for (i, timestamp) in (START_TIME..end_time)
-    //         .step_by(SAMPLING_INTERVAL as usize)
-    //         .enumerate()
-    //     {
-    //         let approximate_value = default_slope * timestamp as f64 + default_intercept;
-    //         mse += (values[i] as f64 - approximate_value).powi(2);
-    //         let approximate_value_with_minimized_mse =
-    //             slope_with_minimum_mse * timestamp as f64 + intercept_with_minimum_mse;
-    //         optimized_mse += (values[i] as f64 - approximate_value_with_minimized_mse).powi(2);
-    //     }
-    //     let mse = mse / values.len() as f64;
-    //     let optimized_mse = optimized_mse / values.len() as f64;
-    //     mse > optimized_mse
-    // }
+    fn can_minimize_mse(error_bound: ErrorBound, values: &[Value]) -> bool {
```

Comment:
Very minor so I accepted the PR and asked @chrthomsen for the final round of review, but the missing punctuation at the end of the comments.

Reply:
Fixed.